src/components/MoneyRequestConfirmationList.js

```diff
@@ -252,6 +261,11 @@ function MoneyRequestConfirmationList(props) {
               shouldCalculateDistanceAmount ? DistanceRequestUtils.getDistanceRequestAmount(distance, unit, rate) : props.iouAmount,
               props.isDistanceRequest ? currency : props.iouCurrencyCode,
           );
+    const formattedTaxAmount = CurrencyUtils.convertToDisplayString(props.transaction.taxAmount, props.iouCurrencyCode);
+
+    const defaultTaxKey = props.policyTaxRates.defaultExternalID;
+    const defaultTaxName = (defaultTaxKey && `${props.policyTaxRates.taxes[defaultTaxKey].name} (${props.policyTaxRates.taxes[defaultTaxKey].value}) • ${translate('common.default')}`) || '';
```

Comment:
There are three occurrences where we have this same logic. Would be good to pack it in a function. NAB, can be done in next PR as this PR has grown big